package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Proxy that handles interactions with pagoPA",
   "main": "index.ts",
   "repository": "https://github.com/teamdigitale/italia-pagopa-proxy",
-  "author": "Sergio Manetta <sergio.manetta@it.ibm.com>",
+  "author": "Federico Feroldi <federico@teamdigitale.governo.it>",
   "license": "MIT",
   "scripts": {
-    "build": "npm-run-all -s generate:proxy-api-models lint build:tsc",
+    "build": "npm-run-all -s generate:proxy-api-models generate:yaml-from-wsdl generate:ts-from-yaml lint build:tsc && cp -r src/wsdl dist/wsdl",
     "build:tsc": "tsc",
-    "clean": "rm -r ./dist",
+    "clean": "rimraf dist && rm -f pagopa_api/yaml-file/*.yaml && rm -f src/types/pagopa_api/yaml-to-ts/*.ts",
+    "generate:ts-from-yaml": "rm -f src/types/pagopa_api/yaml-to-ts/*.ts && rm -f src/types/yaml-to-ts/*.ts && gen-api-models --strict 0 --api-spec ./pagopa_api/yaml-file/FespCdService.yaml --out-dir src/types/pagopa_api/yaml-to-ts/ && gen-api-models --strict 0 --api-spec ./pagopa_api/yaml-file/PagamentiTelematiciPspNodoservice.yaml --out-dir src/types/pagopa_api/yaml-to-ts/",
```

Comment:
I would perhaps split this command in multiple sub commands an call them with `npm-run-all`